userguide/jails.rst

```diff
@@ -250,6 +253,10 @@ a new jail.
    |                           |                   | in :guilabel:`IPv4 Address` does not conflict with an existing address.                                 |
    |                           |                   |                                                                                                         |
    +---------------------------+-------------------+---------------------------------------------------------------------------------------------------------+
+   | NAT                       | checkbox          | Network Address Translation (NAT). Transforms local network IP addresses into a single IP address.      |
+   |                           |                   | Set when the jail shares a single connection to the internet with other systems on the network.         |
```

Comment:
This needs to be expanded, something to clarify how NAT works. "When set, the jail is given an internal IP address and connections are forwarded from the host to the jail."